src/middleware/validateSchema.js

```diff
@@ -29,7 +28,6 @@ module.exports = (shouldUseJoiError = false) => {
 		if (includes(supportedMethods, method) && has(schemas, route)) {
 			// get schema for the current route
 			const schema = get(schemas, route)
```

Comment:
this will also be taken out because the schema will be a parameter being passed by other files